skops/card/_model_card.py

```diff
@@ -373,6 +374,47 @@ def add_metrics(self, **kwargs: str) -> "Card":
             self._eval_results[metric] = value
         return self
 
+    def add_permutation_importances(
+        self, feature_importances, columns, plot_file, plot_name
+    ) -> "Card":
+        """Plots permutation importance and saves it to model card.
+
+        Parameters
+        ----------
+        feature_importances : sklearn.utils.Bunch
+            Output of sklearn.inspection.permutation_importance()
+
+        columns :
+            Column names of the data used to generate importances.
+
+        plot_file :
+            Filename for the plot.
+
+        plot_name :
+            Name of the plot.
+
+        Returns
+        -------
+        self : object
+            Card object.
+        """
+        sorted_importances_idx = feature_importances.importances_mean.argsort()
```

Comment:
Here and below, you still use the variable name `feature_importances`, which should be `permutation_importances`.